components/core/tests/User_tests.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from UserManager import *
+import unittest
+import Models
+
+class User:
+    def __init__(self,userName,email,password,auth):
+        self.userName = userName
+        self.email = email
+        self.password = password
+        self.auth = auth
+    def userName(self):
+        return self.userName
+    def password(self):
+        return self.password
+    def email(self):
+        return self.email
+    def auth(self):
+        return self.auth
+
+class Test(unittest.TestCase):
+
+    def test_check_existing_username(self):
+        self.assertEqual(True,check_user_name('rand'))
```

Comment:
Can you use something like PyCharm's auto formatter to format these files.

Reply:
okay i will do that!